src/content/staking/solo/index.md

```diff
@@ -55,7 +55,7 @@ It also requires very basic hardware setup, and some understanding of minimum re
 <ExpandableCard title="Secure key management" eventCategory="SoloStaking" eventName="clicked secure key management">
 Just like how private keys secure your Ethereum address, you will need to generate keys specifically for your validator. You must understand how to keep any seed phrases or private keys safe and secure.
 
-<a href="/security/">Ethereum security and scam prevention</a>
+<a href="/security/"> Ethereum security and scam prevention</a>
```

Comment:
Why did we add a space here?

```suggestion
<a href="/security/">Ethereum security and scam prevention</a>
```

Reply:
because on the rendered page the link buts up against the preceding text

Reply:
We should add `{' '}` before instead

Reply:
Or a `<br/>` or two to bring it down to a new line

Reply:
Pushed up a change to add {' '}

This addresses the issues above.